canonical/lxd-ui
# Overhaul storage section WD-6471

## Done

- overhaul the storage section following [this design](https://www.figma.com/file/akl8XNqp2K5egFKI3RIfoG/23.10-LXD---ISOs-and-Storage-Volumes?type=design&node-id=359-454&mode=design&t=zkJg5ePsCmTtdPYS-0)
- added pool selector to custom volume creation
- added pool editing

Fixes WD-6471

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check storage section with all subpages and detail pages
    - create pool / update / rename / delete it
    - create volume / update / rename / delete it
    - upload an iso / delete it
